### PR TITLE
remove test states in DISHA API

### PR DESCRIPTION
**Pull request by sravfeyn**
@biyeun

### CODE REVIEW

**Comment by sravfeyn** [comment]
Also @calellowitz

**Review by calellowitz — APPROVED** [review]

**Comment by calellowitz** [comment]
looks good once the merge conflict is fixed

File: custom/icds_reports/reports/disha.py [hunk 1]
```diff
@@ -114,8 +114,7 @@ def build_export_json(self):
 
 
 def build_dumps_for_month(month, rebuild=False):
-    states = AwcLocation.objects.values_list('state_name', flat=True).distinct()
-
+    states = AwcLocation.objects.filter(aggregation_level=1, state_is_test=0).values_list('state_name', flat=True)
```
Comment (mkangia) [inline custom/icds_reports/reports/disha.py]: Why do we need to include `aggregation_level` now?

plus, can we make these levels as constants, 
STATE_LEVEL = 1, DISTRICT_LEVEL = 2?

Comment (sravfeyn) [reply custom/icds_reports/reports/disha.py]: Passing `aggregation_level` is just for query optimisation. Updated state_level const.

**Review by mkangia — APPROVED** [review]

**Pull request merged**